jvb/src/main/resources/reference.conf

```diff
@@ -138,6 +138,33 @@ videobridge {
     #   650pps, so its size in terms of millis is 1024/650*1000 ~= 1575ms.
     send-queue-size=1024
   }
+  load-management {
+    load-measurements {
+      packet-rate {
+        # The packet rate at which we'll consider the bridge overloaded
+        load-threshold = 50000
```

Comment:
How were these numbers computed?  Via testing, or arbitrary?  How would we validate them?

Reply:
The 50000 is an existing number which arrived at for Octo load balancing.  The 40000 was with the 50000 in mind, but arbitrary.